digitalocean/kubernetes/resource_kubernetes_cluster.go

```diff
@@ -121,9 +121,10 @@ func ResourceDigitalOceanKubernetesCluster() *schema.Resource {
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"day": {
-							Type:     schema.TypeString,
-							Optional: true,
-							Computed: true,
+							Type:         schema.TypeString,
+							Optional:     true,
+							Computed:     true,
+							ValidateFunc: validation.StringInSlice([]string{"any", "monday", "tuesday", "wednesday", "thursday", "friday", "saturday", "sunday"}, true),
```

Comment:
I don't have much experience with Go. I see that [Go has no line length limit](https://go.dev/doc/effective_go). So `go fmt` didn't complain or change this.

Should I wrap this to have every value on a separate line for readability?